dev/docker/ballista.dockerfile

```
@@ -49,7 +49,9 @@ ARG PROTOC_VERSION=21.4
 RUN mkdir /tmp/protoc
 WORKDIR /tmp/protoc
 
-RUN export PROTO_ZIP="protoc-${PROTOC_VERSION}-linux-x86_64.zip" && \
+RUN CPU_ARCH=`uname -m` && \
+  if [ $CPU_ARCH=="aarch64" ]; then CPU_ARCH="aarch_64"; fi && \
+  export PROTO_ZIP="protoc-${PROTOC_VERSION}-linux-${ARCH}.zip" && \
```

I think there is a typo here?

```suggestion
  export PROTO_ZIP="protoc-${PROTOC_VERSION}-linux-${CPU_ARCH}.zip" && \
```